docker/test_mcp_search_integration.py

```
@@ -0,0 +1,296 @@
+                    search_results = json.loads(text_content)
+                    if (
+                        "results" in search_results
+                        and len(search_results["results"]) > 0
+                    ):
+                        print(
+                            f"✅ Fallback mode working! Got {len(search_results['results'])} results"
+                        )
+                        return True
+
+        print("⚠️  Fallback test inconclusive")
+        return True
```

Ruff (0.13.3)

Consider moving this statement to an `else` block

(TRY300)